src/vivarium_gates_iv_iron/model_specifications/model_spec.yaml

```diff
@@ -48,7 +48,7 @@ configuration:
         age_end: 54
         include_sex: 'Female'
         pregnant_lactating_women: False
-        exit_age: 56
+#        exit_age: 56
```

Comment:
since age end and exit_age have to be equal, we should make them both 56. It's better to have to initialize a small number of extra 55 and 56 year-olds than to have to track everyone indefinitely after they should have aged out of the simulation, as we have far fewer "useless" person years that way